posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+
+Perhaps even more impactful were the soft skills I developed – my placement year significantly improved my overall confidence and communication skills. Confidence is something that I have personally struggled with and so the benefits of this cannot be overstated.
+
+Finally, my placement year enabled me to “jump start” my graduate role. I am certain that the knowledge of IBM, of enterprise and the network I amassed during this year enabled me to throw myself into the role and progress at a greater momentum than would otherwise have been possible.
+
+image::/img/blog/Billy_Devoxx.jpg
+
+[#ossplacement]
+=== Were you involved in OSS during your placement year? If yes, how did this involvement help you following this placement (i.e. on a personal level, skills, with university, career prospects, etc)?
+
+Most of the work I did was closed source, however I did work on some open source software during placement – the web page to identify Liberty repositories and the sample applications I modernised were open source. There was a great deal I learnt from these projects and the technologies underpinning them - I was also able to work with IBMers who had previously been involved with these projects and it was really useful to be taught the best practises for the samples I was working with, though unfortunately I did not experience direct interaction from the open source community as part of these projects.
+
+[#return]
+=== Why did you want to return after your placement year, to start your career with IBM as a graduate?
+
+I wrote in detail above about what I worked on during my placement, what I learnt and also the incredible additional activities I was fortunate enough to attend during this year. The work at IBM is really varied – as a developer I expected to be coding frequently and learning a vast array of technology, but the role also encapsulated presenting, documentation, DevOps, reporting progress and conceiving ideas at meetings, dealing with customer support issues – and now, writing a blog post! 
+
+I think all of this was enough to have me champing at the bit to return as a graduate, but to extend on that – I met the most amazing people on my placement and that was my primary reason for being so keen to return. Of course, working with top talent presents the opportunity to learn an incredible amount, but I was somewhat surprised by the sheer approachability and kindness shown by these people – there were no stupid questions and despite worrying I would be taking up valuable time or distracting people from more important work I experienced nothing but hospitality and respect. In addition to this, I was matched with a truly fantastic mentor who has coached me throughout my career and been an invaluable source of support and guidance through both my placement year and beyond.
```

Comment:
recommend breaking up longer sentences

I think all of this was enough to have me champing at the bit to return as a graduate. But to extend on that, I met the most amazing people on my placement and that was my primary reason for being so keen to return. Of course, working with top talent presents the opportunity to learn an incredible amount, but I was somewhat surprised by the sheer approachability and kindness shown by these people. There were no stupid questions and despite worrying I would be taking up valuable time or distracting people from more important work, I experienced nothing but hospitality and respect.

Reply:
this one